scripts/instance/mask_rcnn/train_mask_rcnn.py

```diff
@@ -326,7 +326,8 @@ def train(net, train_data, val_data, eval_metric, batch_size, ctx, args):
             net.collect_train_params(),  # fix batchnorm, fix first stage, etc...
             'sgd',
             optimizer_params,
-            update_on_kvstore=(False if args.amp else None), kvstore=kv)
+            update_on_kvstore=(False if args.amp else None),
+            kvstore=(None if args.amp else kv))
```

Comment:
I think 
``` kv = mx.kvstore.create(args.kv_store if not args.amp else 'device')```
is better. The root cause seems to be that nccl kvstore does not support amp

Reply:
Done